tsconfig.json

```diff
@@ -24,10 +24,11 @@
     "skipLibCheck": true,
     "strict": true,
     "strictNullChecks": true,
+    "suppressImplicitAnyIndexErrors": true,
```

Comment:
this feels like overkill (related to dropping the SupportedChainId?) can you just use ts-ignore where you need it instead?